src/consensus/aft/impl/request_message.h

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "consensus/aft/raft_types.h"
+#include "ds/serialized.h"
+#include "kv/kv_types.h"
+#include "message.h"
+
+#include <memory>
+#include <vector>
+
+namespace aft
+{
+// Request messages have the following format.
+#pragma pack(push)
+#pragma pack(1)
+  struct RequestMessageRep : public consensus::ConsensusHeader<RaftMsgType>
+  {
+    RequestMessageRep() = default;
+    RequestMessageRep(
+      aft::NodeId from_node,
+      short command_size_,
+      short cid_,
+      kv::TxHistory::RequestID rid_) :
+      consensus::ConsensusHeader<RaftMsgType>(
+        RaftMsgType::bft_request, from_node),
+      command_size(command_size_),
+      cid(cid_),
+      rid(rid_)
+    {}
+
+    short command_size;
```

Comment:
```suggestion
    uint16_t command_size;
```
Let's use explicit sizes everywhere.